### PR TITLE
Add 2.4.4 as secondary to link-identical-name-equivalent-purpose

### PR DESCRIPTION
**Pull request by WilcoFiers**
I missed a AAA rule that needed a secondary requirement in #1986. 

Need for Call for Review: none, I think. It's a very straight-forward change.

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by HelenBurge — APPROVED** [review]

File: _rules/links-identical-name-equivalent-purpose-b20e66.md [hunk 1]
```diff
@@ -384,7 +388,9 @@ These `span` elements do not have a [semantic role][] of `link`. They are not va
 [matching]: #matching-characters 'Definition of matching characters'
 [presentational roles conflict resolution]: https://www.w3.org/TR/wai-aria-1.1/#conflict_resolution_presentation_none 'Presentational Roles Conflict Resolution'
 [same resource]: #same-resource 'Definition of same resource'
-[sc249]: https://www.w3.org/TR/WCAG21/#link-purpose-link-only 'Success Criterion 2.4.9: Link Purpose (link only)'
+[sc249]: https://www.w3.org/TR/WCAG21/#link-purpose-link-only 'Success Criterion 2.4.9: Link Purpose (Link Only)'
+[sc244]: https://www.w3.org/TR/WCAG21/#link-purpose-in-context 'Success Criterion 2.4.4: Link Purpose (In Context)'
+[sc244]: https://www.w3.org/TR/WCAG21/#link-purpose-link-only 'Success Criterion 2.4.9: Link Purpose (Link Only)'
```
Comment (ajanec01) [inline _rules/links-identical-name-equivalent-purpose-b20e66.md]: I don't think there is need for the 3rd link.

**Review by ajanec01 — DISMISSED** [review]

**Review by carlosapaduarte — DISMISSED** [review]
What Aron requested

File: _rules/links-identical-name-equivalent-purpose-b20e66.md (hunk 1 above)
Comment (WilcoFiers) [reply _rules/links-identical-name-equivalent-purpose-b20e66.md]: Whoop!

Comment (WilcoFiers) [inline _rules/links-identical-name-equivalent-purpose-b20e66.md]:
```suggestion
```

**Review by carlosapaduarte — APPROVED** [review]

**Review by ajanec01 — APPROVED** [review]

**Pull request merged**